cabal.project

```diff
@@ -54,6 +54,21 @@ test-show-details: direct
 -- Always write GHC env files, because they are needed for ghci.
 write-ghc-environment-files: always
 
+constraints:
+  , plutus-ledger-api ^>= 1.31
+  , prettyprinter-configurable ^>= 1.31
+
+allow-newer:
+  -- These are required due to the above.
+  , cardano-ledger-alonzo:plutus-ledger-api
+  , cardano-ledger-alonzo-test:plutus-ledger-api
+  , cardano-ledger-babbage:plutus-ledger-api
+  , cardano-ledger-binary:plutus-ledger-api
+  , cardano-ledger-conway:plutus-ledger-api
+  , cardano-api:plutus-ledger-api
+  , cardano-api:prettyprinter-configurable
+  , cardano-api:base
```

Comment:
Those ones shouldn't be needed right?

Reply:
To build with `ghc-9.10` they are.

Reply:
Are we in a rush for `9.10`? I'd prefer for the CI issues to be resolved in 599.

Reply:
Its not really urgent for anyone other than me. It is significantly more difficult for me to work on X that depends on Y if Y has not been merged to master.